src/Misc/fileTools.ts

```diff
@@ -12,6 +12,7 @@ import { ShaderProcessor } from '../Engines/Processors/shaderProcessor';
 import { ThinEngine } from '../Engines/thinEngine';
 import { EngineStore } from '../Engines/engineStore';
 import { Logger } from './logger';
+import { Tools } from './tools';

```

Comment:
this causes a circular dependency (tools -> fileTools -> tools)

Reply:
I moved the data url logic into fileTools so that it doesn't create a circular import. I then redeclared the functions in tools to maintain backwards compatibility.

@RaananW, I didn't update Babylon.JS's internal references to `Tools.IsBase64` and `Tools.DecodeBase64` to use the new declarations in FileTools. I wasn't sure what the standard practice was there. Should I change all of those references in this PR or keep it minimal?

Reply:
That's perfectly fine. As long as the public functions are all there you can leave it this way. Would be great to just have a deprecated comment for future reference

Reply:
@deprecated docs have been added. Good call 👍